@navikt/core/react/src/provider/theme/AkselTheme.tsx

```diff
@@ -0,0 +1,55 @@
+import cl from "clsx";
+import React, { forwardRef } from "react";
+import { createContext } from "../../util/create-context";
+
+type AkselThemeContext = {
+  appearance: "light" | "dark";
+};
+
+const [ThemeProvider, useAkselTheme] = createContext<AkselThemeContext>({
+  errorMessage: "useAkselThemeProvider must be within AkselThemeProvider",
+  hookName: "useAkselThemeProvider",
+  name: "AkselThemeProvider",
+  providerName: "AkselThemeProvider",
+});
+
+type AkselThemeProps = {
+  children: React.ReactNode;
+  className?: string;
+  hasBackground?: boolean;
+} & AkselThemeContext;
+
+const AkselTheme = forwardRef<HTMLDivElement, AkselThemeProps>(
+  (props: AkselThemeProps, ref) => {
+    const context = useAkselTheme(false);
+
+    const {
+      children,
+      className,
+      appearance = context?.appearance ?? "light",
+      hasBackground: hasBackgroundProp = true,
+    } = props;
+
+    const isRoot = context === undefined;
+
+    const hasBackground =
+      hasBackgroundProp ?? (isRoot || props.appearance !== undefined);
+
+    return (
+      <ThemeProvider appearance={appearance}>
+        <div
+          ref={ref}
+          className={cl("navds-theme", className, {
+            light: appearance === "light",
+            dark: appearance === "dark",
+          })}
```

Comment:
Could we just do this, or could that be a problem? 🤔
```suggestion
          className={cl("navds-theme", className, appearance)}
```

Reply:
Might make it a little obscure for what's happening, but did it for now 👍